contracts/clients-tests/BlsSigner.test.ts

```diff
@@ -1,17 +1,27 @@
+/* eslint-disable camelcase */
 import { ethers as hardhatEthers } from "hardhat";
-import { expect } from "chai";
+import chai, { expect } from "chai";
+import spies from "chai-spies";
 import { ethers, BigNumber } from "ethers";
-import { parseEther, resolveProperties, RLP } from "ethers/lib/utils";
+import {
+  parseEther,
+  resolveProperties,
+  RLP,
+  formatEther,
+} from "ethers/lib/utils";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 
 import {
   Experimental,
   ActionData,
   BlsWalletWrapper,
   NetworkConfig,
+  MockERC20__factory,
 } from "../clients/src";
 import getNetworkConfig from "../shared/helpers/getNetworkConfig";
 
+chai.use(spies);
```

Comment:
Missed this on last PR, but you can remove these `chai.use` statements from individual test files and put it in https://github.com/web3well/bls-wallet/blob/main/contracts/clients/test/init.ts to make it usable in any test in the test suite.

Reply:
Yeah that looks much cleaner. Although since the tests are currently in their own directory `./clients-tests` outside of `./clients`, it won't get picked up. `chai-spies` is currently installed in the contracts directory so I also get a `no-extraneous-import` error when I add the imports to `init.ts`. Assume that means I can't make this change?

I'm picking up the integrations tests issue after my next PR so can add a todo to think about this at that point if that works?

Reply:
Ah yep forgot about that, yeah can do this when we setup automated integration tests.

Good as is.